aviatrix/resource_aviatrix_firewall_instance.go

```diff
@@ -199,6 +199,14 @@ func resourceAviatrixFirewallInstance() *schema.Resource {
 				Computed:    true,
 				Description: "Management Public IP.",
 			},
+			"tags": {
+				Type:        schema.TypeMap,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				Optional:    true,
+				Default:     nil,
```

Comment:
This is unnecessary, Default is of type interface{} and the default value for interface{} is already nil.